lib/os/path.py

```diff
@@ -35,3 +36,33 @@ def join(*paths):
   if result and not paths[-1]:
     result += '/'
   return result
+
+
+def normpath(path):
```

Comment:
Sorry for noticing this late, could you put the methods in this module in alphabetical order please? Also the tests in the test file.